spec/exercise_spec.rb

```diff
@@ -0,0 +1,124 @@
+require "rails_helper"
+
+RSpec.describe Exercise, type: :model do
+  describe "create_exercise" do
+    let(:user) { User.create(username: "Sean", password: "1234") }
+    let(:workout_id) { 1 }
+    let(:sets) { 1 }
+    let(:blueprint) { Blueprint.create(name: "Shoulder Abs") }
+
+    before do
+      increase_rep_by_1 =
+        Exercise.create(
```

Comment:
I'll sometimes move the basic artifact creation to a different file to help tests be readable. Not really necessary at this point, just figured I'd mention it 🤷